pkg/controllers/config/configsync_test.go

```diff
@@ -0,0 +1,136 @@
+package config
+
+import (
+	"strings"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestTranslateConfigMap(t *testing.T) {
+	format.MaxDepth = 100
+	format.TruncatedDiff = false
+
+	tc := []struct {
+		name                string
+		source              string
+		target              string
+		isMultiAZDeployment bool
+		errMsg              string
+	}{
+		{
+			name: "Config with unsupported secret-namespace override",
+			source: `[Global]
+secret-namespace = foo
+secret-name = openstack-credentials`,
+			errMsg: "'[Global] secret-namespace' is set to a non-default value",
+		}, {
+			name: "Config with unsupported secret-name override",
+			source: `[Global]
+secret-namespace = kube-system
+secret-name = foo`,
+			errMsg: "'[Global] secret-name' is set to a non-default value",
+		}, {
+			name: "Config with unsupported kubeconfig-path override",
+			source: `[Global]
+secret-namespace = kube-system
+secret-name = openstack-credentials
+kubeconfig-path = https://foo`,
+			errMsg: "'[Global] kubeconfig-path' is set to a non-default value",
+		}, {
+			name:   "Empty config",
+			source: "",
+			target: `[Global]
+use-clouds  = true
+clouds-file = /etc/openstack/secret/clouds.yaml
```

Comment:
We've changed the clouds file to be in `/etc/kubernetes/secret/clouds.yaml` to retain the previous location. Could you update the tests as well?

Reply:
Good spot. Done